spec/features/user_management_spec.rb

```diff
@@ -0,0 +1,74 @@
+require_relative '../factories/user'
+
+feature 'user sign up' do
+  scenario 'I can sign up as a new user to Chitter' do
+    user = build :user
+    sign_up(user)
+    expect(page).to have_content "Welcome, #{user.email}"
+  end
+
+  scenario 'with a password that does not match' do
+     user = create :user
+     user.password_confirmation = 'wrong'
+     sign_up(user)
+     expect(page).to have_content 'Password does not match the confirmation'
+   end
```

Comment:
Lint/BlockAlignment: `end` at 15, 3 is not aligned with `scenario 'with a password that does not match' do` at 10, 2